includes/compat/class-payment.php

```diff
@@ -150,8 +150,21 @@ public function get_post_metadata( $value, $object_id, $meta_key, $single ) {
 			'_edd_payment_total',
 			'_edd_payment_tax',
 			'_edd_payment_number',
+			'_edd_sl_upgraded_payment_id', // EDD SL
+			'_edd_sl_is_renewal', // EDD SL
+			'_edds_stripe_customer_id', // EDD Stripe
```

Comment:
@ashleyfae If these whitelists are going to be the same (`get_` and `update_`), should we add a private method to this class so we only have to maintain one list, and document one filter?

Reply:
Good idea @spencerfinnell ! I'll add that tomorrow.